engine/src/timeperiod.cc

```diff
@@ -109,41 +109,6 @@ bool timeperiod::operator!=(timeperiod const& obj) noexcept {
   return !(*this == obj);
 }
```

Comment:
why remove it, it may be usefull to debug in future

Reply:
I prefer to rewrite them when needed.

Reply:
I just moved these two operators to configuration.